app/datatables/organization_datatable.rb

```diff
@@ -0,0 +1,29 @@
+class OrganizationDatatable < ApplicationDatatable
+  def view_columns
+    @view_columns ||= {
+      organization: {source: "Officer.organization", },
+      num_employees: {orderable: false, searchable: false}
+    }
+  end
+
+  def data_record(record)
+    {
+      url: organization_url(record.organization.parameterize),
+      organization: record.organization,
+      num_employees: record.num_employees
+    }
+  end
+
+  def get_raw_records
+    Officer.distinct.select("organization, count(employee_id) as num_employees").
+      group(:organization).where.not(organization: nil).order("count(employee_id) desc")
+  end
+
+  def records_total_count
+    fetch_records.size.size
```

Comment:
.size.size? how does this work? I'm playing with it, and `fetch_records.size.size` does appear to work and gives me 153.

`fetch_records.count` gives me a weird ActiveRecord::StatementInvalid error. I'm guessing `count` doesn't play well with custom select clauses?

Maybe this shouldn't use the Officer model, it's kind of awkward, right?
```
# rough untested idea:
Officer.connection.select_all("select organization, count(*) as count from officers group by organization").map do |r|
  OpenStruct.new(organization: r["organization"], num_employees: r["count"]
end
```

Reply:
Ah, yeah - this was my workaround for the fact that `fetch_records.count` produces an invalid SQL query and that `fetch_records.size` returns a hash mapping organization names to employee counts rather than an array of records.

That general idea is definitely better! My bad for trying to get too clever with the ORM 🙂

Reply:
yeah, sometimes it's hard to know when we're being clever, and when we've broke it 😝